loopd/swapclient_server.go

```diff
@@ -433,11 +438,16 @@ func (s *swapClientServer) LoopIn(ctx context.Context,
 	np2wshAddress := swapInfo.HtlcAddressNP2WSH.String()
```

Comment:
Can be moved inside `if in.ExternalHtlc`, because only used there?

Reply:
done